src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml.cs

```diff
@@ -185,5 +239,13 @@ public async Task<IActionResult> OnPostToggleApprovalAsync(string id, string rev
             await _manager.ToggleApprovalAsync(User, id, revisionId);
             return RedirectToPage(new { id = id });
         }
+        public Dictionary<string, string> GetRoutingData(string diffRevisionId = null, bool? showDocumentation = null, bool? showDiffOnly = null)
+        {
+            var routingData = new Dictionary<string, string>();
+            routingData["diffRevisionId"] = diffRevisionId;
+            routingData["doc"] = (showDocumentation ?? showDocumentation) == true ? "true" : "false";
```

Comment:
Why do we have this expression: `showDocumentation ?? showDocumentation`

Reply:
(showDocumentation ?? false).ToString()?

Reply:
Thanks. I will make this change. Not sure what exactly I was thinking when wrote x ?? x. Thanks for catching this.